boilerplate/app/utils/validate.ts

```diff
@@ -70,7 +70,7 @@ export interface ValidationErrors {
  * @param rules The rules to apply.
  * @param data The object to validate.
  */
-export function validate(rules: ValidationRules, data: {}): ValidationErrors {
+export function validate(rules: ValidationRules, data: Record<string, unknown>): ValidationErrors {
```

Comment:
pretty

Reply:
That's straight out of the lint error message, FYI:
```
Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead  @typescript-eslint/ban-types
```